src/OrchardCore.Modules/OrchardCore.ContentFields/ContentItemDisplayExtensions.cs

```diff
@@ -0,0 +1,39 @@
+using System.Collections.Generic;
+using System.Threading.Tasks;
+using Microsoft.Extensions.Localization;
+using OrchardCore.ContentLocalization;
+using OrchardCore.ContentManagement;
+using OrchardCore.ContentManagement.Routing;
+
+namespace OrchardCore.ContentFields
+{
+    public static class ContentItemDisplayExtensions
+    {
+        public static async Task<string> GetContentItemRouteAndCulture(this ContentItem contentItem, IContentManager contentManager, IStringLocalizer S)
+        {
+            var displayText = string.Empty;
+            var additionalParts = new List<string>();
+
+            // add culture to display text
+            var cultureAspect = await contentManager.PopulateAspectAsync<CultureAspect>(contentItem);
+            if (cultureAspect.HasCulture)
+            {
+                additionalParts.Add($"{S["Culture"]}: {cultureAspect.Culture}");
+            }
+
+            // add autoroute path to display text
+            var routeAspect = await contentManager.PopulateAspectAsync<RouteHandlerAspect>(contentItem);
```

Comment:
don't use this aspect get the path. It's not pathbase / tenant aware. this is used for contained routing.
You need the url helper, and `ContentItemMetadata.DisplayRouteValues` to generate a valid path.